web-admin/src/routes/[organization]/[project]/+page.ts

```diff
@@ -0,0 +1,16 @@
+import { featureFlags } from "@rilldata/web-common/features/feature-flags.js";
+import { redirect } from "@sveltejs/kit";
+import { get } from "svelte/store";
+
+export const load = async ({ params: { organization, project }, parent }) => {
+  // Wait for the feature flags to load
+  await parent();
+
+  // NOTE: in the future, we'll use user-level `ai` permissions to determine access to the chat page.
+  const chatEnabled = get(featureFlags.chat);
```

Comment:
Would we want some kind of user setting as well? I doubt every user would want to land on chat if admin enables this feature.